api/StorageManager.json

```diff
@@ -5,34 +5,56 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/StorageManager",
         "support": {
           "webview_android": {
-            "version_added": null
+            "version_added": "48"
           },
           "chrome": {
-            "version_added": null
+            "version_added": "48"
           },
           "chrome_android": {
-            "version_added": null
+            "version_added": "48"
           },
           "edge": {
             "version_added": null
           },
           "edge_mobile": {
             "version_added": null
           },
-          "firefox": {
-            "version_added": null
-          },
+          "firefox": [
+            {
+              "version_added": "57"
+            },
+            {
+              "version_added": "51",
+              "version_removed": "57",
+              "notes": "See <a href='https://bugzil.la/1304966'>Bug 1304966</a> and <a href='https://bugzil.la/1399038'>Bug 1399038</a>.",
```

Comment:
I don't think we routinely capitalize "bug"

Reply:
Thanks! I fixed the capitalization now and added issue #2509 to potentially check this as part of linting.